styles.css

```diff
@@ -0,0 +1,145 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* Media screen for mobiles with minimum width of 375px */
+@media screen and (min-width: 375px) {
+  body {
+    background-image: url(./images/background.png);
+    background-size: contain;
+  }
```

Comment:
- [ ] Kindly consider adding `background-repeat: no-repeat;` because the background image is repeating. 
- [ ] Also consider taking the background-image for the mobile screen here because you are using a background-image here which should be used for the bigger displays above the `768px` mark. so kindly download and use the appropriate mobile-screen background image here.